### PR TITLE
Add operator helm chart

### PR DESCRIPTION
**Pull request by michaeljguarino**
just a 1 replica deployment with variables for deploy token and url

### CODE REVIEW

File: helm/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: deployment-operator
+description: creates a new instance of the plural deployment operator
+type: application
+version: 0.9.1
```
Comment (maciaszczykm) [inline helm/Chart.yaml:5]: Should we update to 0.1.0 or something like that?

File: helm/values.yaml.liquid [hunk 2]
```diff
@@ -0,0 +1,4 @@
+secrets:
+    deployToken: {{ configuration.deployToken }}
```
Comment (maciaszczykm) [inline helm/values.yaml.liquid:2]: Not sure if this will have any impact, but indentation is 4 spaces instead of 2.

Comment (maciaszczykm) [inline helm/values.yaml.liquid:2]: Indentation is 4 spaces instead of 2.

File: helm/Chart.yaml (hunk 1 above)
Comment (michaeljguarino) [reply helm/Chart.yaml:5]: We'll probably have a lot of prereleases for it, we'll eventually go to a 0.x version when it's stable

**Review by maciaszczykm — APPROVED** [review]

**Review by zreigz — APPROVED** [review]

**Review by floreks — APPROVED** [review]

**Pull request merged**